packages/paste-website/src/pages/blog/2023-11-08-paste-newsletter.mdx

```diff
@@ -0,0 +1,214 @@
+
+### Editable Code Block
+
+<ResponsiveImage src={EditableCodeBlock} alt="Paste Editable Code Block component example" />
+
+We’ve built an [Editable Code Block](/components/editable-code-block) that allows basic text input features for code-editing experiences on the web. It’s built on top of the Paste [Code Editor library](/core/libraries/code-editor), which wraps the Monaco Editor used by Visual Studio Code.
+
+### Error pattern
+
+<Box width="100%" display="flex" justifyContent="center">
+  <Box maxWidth="600px">
+    <ResponsiveImage src={ErrorPattern} alt="Do/Dont example from the error state pattern documentation" />
+  </Box>
+</Box>
+
+**Megan Allison** contributed yet another pattern: the [error state pattern](/patterns/error-state)! It includes extensive guidelines for notifying a user that something went wrong. Read the error pattern documentation for guidance on how to compose the message itself, which components to use for each specific error case, a comprehensive list of recommended phrasings, and accessibility guidance.
+
+### Additional updates
+
+We’ve just completed our annual round of roadmap reviews with product teams to help shape the future of our roadmap. Thanks to the Segment, Flex, Messaging, Voice, Web Platform, Commerce, Onboarding, Phone Numbers, and Brand teams for sharing your upcoming initiatives!
+
+#### New
+
+- Added a set of delivery status [icons](/components/icon).
+- Added the [confirmation pattern](/patterns/confirmation) (also contributed by Megan Allison). A confirmation ensures a user understands what will happen if they take an action that’s destructive, complex, or has an unexpected result.
+
+#### Improvements
+
+- Improved the [In Page Navigation](/components/in-page-navigation) component with design updates and additional documentation.
+
+## ✨ Pastemates™ spotlight
+*Pastemates are individuals and teams who have been critical contributors to Paste, through advocating for adoption within their products and channeling important feedback and contributions back into the system.*
+
+<Box width="100%" display="flex" justifyContent="center">
+  <Box maxWidth="600px">
+    <ResponsiveImage src={Pastemate1} alt="Pablo Deeleman" />
+  </Box>
+</Box>
+
+In this edition of Pastemates, we're spotlighting a mate from the Web Platform team, Pablo Deeleman! Pablo lives the Twilio Builder spirit, building complex prototypes with just out-of-the-box Paste components. During his time on the Event Streams team, he pushed the boundaries of what could be built with Paste "as is":
+
+[Watch the Event Streams demo video](https://www.loom.com/share/6b20722099174c7cbe8932965f74d1b9?sid=14225154-8f17-471d-8d35-a51fbda5f943)
```

Comment:
suggestion: remove this link entirely from the public site. Should probably be password protected

Reply:
done